packages/interface/src/GraphQLLanguageService.js

```diff
@@ -107,7 +107,11 @@ export class GraphQLLanguageService {
       ];
     }
 
-    if (!schemaPath) {
+    const schema = await this._graphQLCache
```

Comment:
Would it be possible not to `await` for the schema to be created here and below? The reason why we sanity-check the path (and the endpoint in addition according to this PR) is not to spend any unnecessary time to compute the schema if it's not necessary. This usually isn't a problem unless there's a validation error in the schema creation process, in which this will execute and fail every time.

Reply:
I think the best way to prevent thrashing is to "cache" both successful schemas and failures (both from disk and network) and disallow re-fetching until expiry. In which case we would keep `await getSchema` but trust that it will fail fast. However this feels beyond the scope of the current change – would you be happy to merge this change without it?

Reply:
I think we should merge with the behavior as is and file a ticket to address this failure mode next. Existing users won’t be impacted since there is currently no online schema support anyway. If they have have broken disk schema this await will fail fast.